app/models/conversation-part.js

```diff
@@ -4,6 +4,7 @@ import { belongsTo } from 'ember-data/relationships';
 
 export default Model.extend({
   body: attr(),
+  closedAt: attr(),
```

Comment:
There is no `closedAt` on the model

The component's `closedAt` property can be set to the model's `insertedAt` or `updatedAt` as we discussed. You already do this on line 16 of `app/templates/components/conversations/conversation-thread.hbs`.